setup.py

```diff
@@ -44,10 +44,10 @@
     ],
     keywords=["mender", "OTA", "updater"],
     packages=setuptools.find_packages(where="src"),
-    install_requires=["cryptography", "requests"],
+    install_requires=["cryptography", "requests", "msgpack", "websockets"],
     entry_points={"console_scripts": ["mender-python-client=mender.mender:main"]},
     package_dir={"": "src"},
-    python_requires=">=3.6",
+    python_requires=">=3.9",
```

Comment:
Is this still a requirement?

Reply:
If we don't support a 'user' option, and we don't, then this is not a requirement anymore.
Will be reverted to 3.6